testing/smoke-test/build-pack.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Smoke test environment setup script
+
+cu_p=packages/core-util
+d_p=packages/dashboard
+rc_p=packages/react-components
+sw_p=packages/source-iotsitewise
+
+npm run build:dashboard
+
+npm pack -w @iot-app-kit/core
+
+c_tb=$(find . -name 'iot-app-kit-core-*.tgz' -type f -exec basename {} \; | head -n 1)
+
+find $cu_p $d_p $rc_p $sw_p -type d -maxdepth 0 -exec cp $c_tb {} \;
+
+npm pkg -w @iot-app-kit/core-util set files[]="$c_tb"
+
+npm pack -w @iot-app-kit/core-util
+
+cu_tb=$(find . -name 'iot-app-kit-core-util-*.tgz' -type f -exec basename {} \; | head -n 1)
+
+find $d_p $rcp $sw_p -type d -maxdepth 0 -exec cp $cu_tb {} \;
+
+npm pkg -w @iot-app-kit/source-iotsitewise set files[]="$c_tb"
+npm pkg -w @iot-app-kit/source-iotsitewise set files[]="$cu_tb"
+
+npm pack -w @iot-app-kit/source-iotsitewise
+
+sw_tb=$(find . -name 'iot-app-kit-source-iotsitewise-*.tgz' -type f -exec basename {} \; | head -n 1)
+
+find $d_p $rc_p -type d -maxdepth 0 -exec cp $sw_tb {} \;
+
+npm pkg -w @iot-app-kit/react-components set files[]="$c_tb"
+npm pkg -w @iot-app-kit/react-components set files[]="$cu_tb"
+npm pkg -w @iot-app-kit/react-components set files[]="$sw_tb"
+
+npm pack -w @iot-app-kit/react-components
```

Comment:
Do we need to pack all of these? Or is this setup so we can include further smoke testing in the future?